### PR TITLE
Fix devcontainer

### PR DESCRIPTION
**Pull request by matsonj**
Not sure how to get tasks to work... the task sequencing doesn't seem like it works, as it is waiting for the terminal to return that the task has completed before executing the next task.

### CODE REVIEW

File: .vscode/tasks.json [hunk 1]
```diff
@@ -9,43 +9,57 @@
...
         "presentation": {
           "panel": "shared",
-          "close": false,
+          "close": false
         }
+      },
+      {
+        "label": "duckdbt!",
+        "dependsOrder": "sequence",
+        "dependsOn": [
```
Comment (matsonj) [inline .vscode/tasks.json:50]: This is the problematic bit, each does not start until the previous task "completes" but they are services and thus do not complete.

**Comment by matsonj** [comment]
so its mergeable but the tasks don't work. IDK if that is worth fixing or should just orchestrate w/make?

File: .vscode/tasks.json (hunk 1 above)
Comment (jwills) [reply .vscode/tasks.json:50]: ack, gotcha

**Comment by jwills** [comment]
yeah doing it with make seems the way to go, so much for my attempt to make tasks work nbd

**Pull request merged**